packages/eslint-plugin/docs/rules/no-unsafe-enum-comparison.mdx

```diff
@@ -9,7 +9,9 @@ import TabItem from '@theme/TabItem';
 >
 > See **https://typescript-eslint.io/rules/no-unsafe-enum-comparison** for documentation.
 
-The TypeScript compiler can be surprisingly lenient when working with enums. String enums are widely considered to be safer than number enums, but even string enums have some pitfalls. For example, it is allowed to compare enum values against literals:
+The TypeScript compiler can be surprisingly lenient when working with enums.
+While overt unsafety problems with enums were [resolved in TypeScript 5.0](https://devblogs.microsoft.com/typescript/announcing-typescript-5-0/#all-enums-are-union-enums), some logical pitfalls remain permitted.
```

Comment:
Nit: TIL _"unsafety"_ is a word. It's not in my Mac+Chrome default dictionary though 😄. And I think _"safety problems"_ is a more common phrase. 

```suggestion
While overt safety problems with enums were [resolved in TypeScript 5.0](https://devblogs.microsoft.com/typescript/announcing-typescript-5-0/#all-enums-are-union-enums), some logical pitfalls remain permitted.
```